cli/flox/doc/flox-containerize.md

```diff
@@ -66,6 +66,63 @@ similar to `flox activate --`.
 ./include/general-options.md
 ```
 
+# MANIFEST CONFIGURATION
+
+Configuration for the container image produced by `flox containerize` may be specified in a `[containerize.config]` table in the environment manifest.
+
+> **Warning:**
+> `containerize.config` is **experimental**,
+> and its behaviour is subject to change
+
+The following options from the OCI spec are supported, specified in `kebab-case` rather than `PascalCase`:
```

Comment:
I wouldn't really say it's from the OCI spec, since [`dockerTools.streamLayaredImage` in nixpkgs](https://nixos.org/manual/nixpkgs/stable/#ssec-pkgs-dockerTools-streamLayeredImage) is still referencing the Docker Image specs. With OCI spec I think [this](https://github.com/opencontainers/image-spec/blob/main/config.md). Now, I believe the OCI spec is a fork from the Docker Image spec, and they might be identical. I haven't check t.b.h.

Reply:
tl;dr from https://www.notion.so/floxdev/Merge-strategy-15a744c8e316800c9a0ceb6f4ee19794 is that there isn't much difference between the two. I think it's clearer to reference OCI than Docker though, because the user may not be interfacing with Docker at all (e.g. `--runtime podman`).